src/main/java/org/skriptlang/skript/bukkit/input/elements/expressions/ExprCurrentInputKeys.java

```diff
@@ -0,0 +1,77 @@
+package org.skriptlang.skript.bukkit.input.elements.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerInputEvent;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.input.InputKey;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Player Input Keys")
+@Description("Get the current input keys of a player.")
+@Examples("broadcast \"%player% is pressing %current input keys of player%\"")
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.21.2+")
+public class ExprCurrentInputKeys extends PropertyExpression<Player, InputKey> {
+
+	private static final boolean SUPPORTS_TIME_STATES = Skript.classExists("org.bukkit.event.player.PlayerInputEvent");
+
+	static {
+		register(ExprCurrentInputKeys.class, InputKey.class, "[current] (inputs|input keys)", "players");
+	}
+
+	@Override
+	public boolean init(Expression<?>[] expressions, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		setExpr((Expression<? extends Player>) expressions[0]);
+		return true;
+	}
+
+	@Override
+	protected InputKey[] get(Event event, Player[] source) {
+		Player eventPlayer = null;
+		if (SUPPORTS_TIME_STATES && getTime() == EventValues.TIME_NOW && event instanceof PlayerInputEvent inputEvent)
+			eventPlayer = inputEvent.getPlayer();
+
+        List<InputKey> inputKeys = new ArrayList<>();
```

Comment:
```suggestion
		List<InputKey> inputKeys = new ArrayList<>();
```
some spaces have made their way in